versionist.conf.js

```diff
@@ -0,0 +1,59 @@
+var execSync = require('child_process').execSync;
+
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
```

Comment:
Nope. I've just been copy/pasting this file around from repo to repo for a while now.